R/install.R

```diff
@@ -371,36 +371,6 @@ build_cmdstan <- function(dir,
   )
 }
 
-# Removes files that are used to simplify switching to using threading, opencl or mpi.
-clean_compile_helper_files <- function() {
```

Comment:
This has not been needed for some time, we just haven't cleaned it up.

Reply:
Awesome, always nice to delete a bunch of code!